### ISSUE TITLE
Dump Inquiry

### ISSUE THREAD

**Issue by AmericanY** [sqlmapproject/sqlmap#2784]
Hi Guys,

around 3 days till now i couldn't found any fast solution for my case so i decided to write here maybe someone have a solution for me better than the one i use which is very odd

Basically there's MySQL database table include 50K entry and after i applied my query with conditions the dump data is 10K

my issue here that the dumped data included some entry include Unicode such as `? `which occurred because the MySql server were heavy during the dump.

let's say on the dumped 10K row number 1100 and 1200 is missed with Unicode and need to dumped again. 

so the only way i use now is to submit query by query 

LIMIT 1099, 1 in order to get row number 1100 then apply LIMIT 1199, 1 to get row number 1200 which is need to done command by command !

so are there way to use limit multiple time in one query ? or are there a way to pick up the entry by the row number directly since i do have a list of row numbers which is missed !

**Comment by stamparm** [comment]
Is this sqlmap related? You are mentioning `LIMIT M,N` without stating whether you've used sqlmap and how in the first place?

If sqlmap related, you can't used multiple "limit" at the same time. You should combine something like `--fresh-queries --start=M --stop=N` for re-dumping rows M to N in a run

**Comment by AmericanY** [comment]
Indeed i used salamp to dump the data using —sql-query option, therefor there’s many dumped data include unicodes (?) etc... so I already doubled checked the server side by redump some of missed data and I confirmed they are correct on the server side.

Using start and stop which is same as LIMIT usage gonna be useful for data which is in range but will not make sense for single random data in case if there’s multi rows need to be redumped.

Anyway I went ahead and assigned all row numbers minus one number with LIMIT ( line by line command ) then  with small bash script include line by line command  ( command must be configured with sqlmap answers to not pause the stdout during run such as follow=Y ) to redump all missed data once again by LIMIT

Close the topic.

Thanks
